docs/QbeastFormat.md

```diff
@@ -145,19 +145,19 @@ A closer look to the `qb.revision.$number`:
 
 In Revision, you can find different information about the tree status and configuration:
 
-| Term                            | Description                                                                                     |
-|---------------------------------|-------------------------------------------------------------------------------------------------|
-| `timestamp`                     | The time when the revision was created.                                                         |
-| `tableID`                       | The identifier of the table that the revision belongs to.                                       |
-| `desiredCubeSize`               | The cube size from the option `cubeSize`.                                                       |
-| `columnTransformers`            | The metadata of the different columns indexed with the option `columnsToIndex`.                 |
-| `columnTransformers.columnName` | The name of the column.                                                                         |
-| `columnTransformers.dataType`   | The data type of the column.                                                                    |
-| `transformations`               | Contains information about the **space** of the data indexed by column.                         |
-| `transformations.className`     | The name of the class that implements the transformation.                                       |
-| `transformations.minNumber`     | The minimum value.                                                                              |
-| `transformations.maxNumber`     | The maximum value.                                                                              |
-| `transformations.nullValue`     | The value that represents the null in the space.                                                |
+| Term                            | Description                                                                     |
+|---------------------------------|---------------------------------------------------------------------------------|
+| `timestamp`                     | The time when the revision was created.                                         |
+| `tableID`                       | The identifier of the table that the revision belongs to.                       |
+| `desiredCubeSize`               | The cube size from the option `cubeSize`.                                       |
+| `columnTransformers`            | The metadata of the different columns indexed with the option `columnsToIndex`. |
+| `columnTransformers.columnName` | The **name of the column**.                                                     |
+| `columnTransformers.dataType`   | The **data type of the column**.                                                |
+| `transformations`               | Contains information about the **space mapping** of the data indexed by column. |
```

Comment:
What are these transformations exactly?

Reply:
Transformations are the functions that map the **real values of a column indexed into the [0.0, 1.0) space.**

Reply:
Updated the description!

Reply:
Cool, why is is "transformations" plural, is it because there is going to be one for each column domain?

Reply:
Correct 🥇